packages/filesystem/src/typings/nsfw/index.d.ts

```diff
@@ -1,40 +1,32 @@
-/*---------------------------------------------------------------------------------------------
```

Comment:
We should have typings in one place. Would it be possible to get rid of this file?

Reply:
I wasn't able to have them share the same file.  You can try if you want.

Reply:
Note that I'd like to work on getting them upstream (`@types/nsfw`) in the DefinitelyTyped repo, to avoid having to maintain it here.